terraform/backend_uk.tf

```diff
@@ -43,6 +43,10 @@ module "backend_uk" {
         name  = "CDS"
         value = "true"
       },
```

Comment:
I think we can just delete the env var and move to having all places serve these paths